uui/components/overlays/Tooltip.tsx

```diff
@@ -1,16 +1,18 @@
 import { withMods } from '@epam/uui-core';
-import { Tooltip as uuiTooltip, TooltipProps as UuiTooltipProps } from '@epam/uui-components';
+import * as UuiComponents from '@epam/uui-components';
```

Comment:
```suggestion
import * as uuiComponents from '@epam/uui-components';
```
use small letters for namespaces